### ISSUE TITLE
Undefined variable

### ISSUE THREAD

**Issue by wch1996** [jshermeyer/RFSR#2]
Dear Jshermeyer,
Thank for your hard working.
I have a question in 320th line:
Train_TrainSet, Train_TestSet, Target_TrainSet, Target_TestSet = train_test_split(Trains[0], Targets[0], test_size=0.1, train_size=0.1)
'Trains' and 'Targets' are not defined,Is this program unnecessary?

**Comment by jshermeyer** [comment]
Yes I think this is actually a redundancy I forgot to remove.  Uploaded the updated code with this removed to master.
